src/app/landing/LandingPage.jsx

```diff
@@ -57,34 +64,90 @@ class LandingPage extends Component {
               <h3>How it works</h3>
               <h2>Search for what you need</h2>
               <p className="text-lighter">
-                GoGetta has New York’s most up-to-date service information on food, personal care, shelter, and clothing.
+                GoGetta has New York’s most up-to-date service information on
+                food, personal care, shelter, and clothing.
               </p>
             </div>
 
             <picture>
+              {/* desktop jpg */}
               <source
-                className="OnboardingImage"
-                media="(min-aspect-ratio: 2/3)"
-                srcSet="img/landing_page/map_tablet.png"
+                media="(min-width: 1224px)"
+                srcSet="/img/landing_page/step1_desktop1x.jpg 1x,
+                /img/landing_page/step1_desktop2x.jpg 2x"
+                type="image/jpeg"
+              />
+              {/* desktop webp */}
+              <source
+                media="(min-width: 1224px)"
+                srcSet="/img/landing_page/step1_desktop1x.webp 1x,
+              /img/landing_page/step1_desktop2x.webp 2x"
+                type="image/webp"
+              />
+              {/* mobile jpg */}
+              <source
+                media="(max-width: 1223px)"
+                srcSet="/img/landing_page/step1_mobile2x.jpg"
```

Comment:
I'm still a bit confused about the sizes: If I understand correctly, currently all devices with a size under 1224 _CSS_ pixels, i.e. _all_ phones, will get this version, which I assume is the one Di provided that is e.g. 1224x817 for this step1 picture.
That seems a little small even for old phones with just 2x density (even e.g. an iPhone 6 has a size of 750x1334 hardware pixels, which, if the picture takes up say 60% of the viewport height - 800px - is just barely small enough not to require stretching).
Certainly if we're talking about newer devices, with better density or larger size than an iPhone 6, the picture won't be as sharp as it could be.

Of course, this isn't a huge deal (the primary users we care about presumably don't have the most cutting-edge devices), but the question is why we wouldn't look at the density for the mobile case too, as you already do for desktop?
In a way, it seems referring to the images as "mobile2x", "desktop1x", and "desktop2x" is somewhat misleading, because in reality a high-density phone could benefit from a larger file size just as much as a normal-density desktop.

I suppose one reason could be that even for high-res phones, there's more of a likelihood that they're using a mobile data connection, meaning bandwidth is a concern, whereas for desktops that's less likely to be the case (although anyone with a high-res phone is more likely to have Unlimited Data anyway...).

But on that note, my understanding is that there's already a way to let the browser make the decision of which picture to choose, based on all available information, including bandwidth: Just use `srcset` with `w` values specifying the file sizes (and pass `sizes`), instead of relying on `media` and density breakpoints.
I haven't personally used any of this before, so I could be misunderstanding, but from what I see online this is supposed to allow the browser to make informed decisions, and also saves you the complexity of dealing with multiple sources + srcset options (of course, multiple sources will still be needed for webp vs. jpg).

It isn't a huge deal one way or the other, so it's also fine to leave things as-is if you prefer, but if it could lead to simplifying the code, providing crisper images on many devices, while keeping the same assets with the same sizes, I figured it's at least worth considering.

Reply:
I used Di’s retina mobile images and further compressed the jpgs/ webps so that they would be small enough for use on both kinds of devices and high quality enough that users (hopefully) shouldn’t notice unless they’re looking very closely.

Reply:
Alright. I suspect for people on a high-end phone it might not feel as sharp as it could, but that's not really the demographic we care about, so that's fine.